Author: Tianon Gravi <admwiggin@gmail.com>
Description: socket activation + systemd 208 made permission issues, so this is a temporary patch-out of socket activation
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=752555

diff --git a/contrib/init/systemd/docker.service b/contrib/init/systemd/docker.service
index eeadd6f..ebbb7c9 100644
--- a/contrib/init/systemd/docker.service
+++ b/contrib/init/systemd/docker.service
@@ -1,12 +1,11 @@
 [Unit]
 Description=Docker Application Container Engine
 Documentation=http://docs.docker.com
-After=network.target docker.socket
-Requires=docker.socket
+After=network.target
 
 [Service]
 EnvironmentFile=/etc/default/docker
-ExecStart=/usr/bin/docker -d -H fd:// $DOCKER_OPTS
+ExecStart=/usr/bin/docker -d $DOCKER_OPTS
 LimitNOFILE=1048576
 LimitNPROC=1048576
 
